instrumentation/aws-sdk/aws-sdk-1.11/library-instrumentor/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
```

Comment:
Can you call the new projects / artifacts here `-autoconfigure` instead of `-instrumentor`? It's sort of a naming scheme we have in OTel.